CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v6.1.2
```

Comment:
do we need a new version even though only test related things changed? 🤔

Reply:
i'm ambivalent. any strong opinions? @ykitamura-mdsol ?

Reply:
same here, but since this PR is just updating test codes, probably we might not need a new version...

Reply:
actually ended up bumping minor version as we're dropping two old ruby versions and it seems like we've bumped minor for that in the past: https://github.com/mdsol/mauth-client-ruby/blob/master/CHANGELOG.md#v410